.github/workflows/avm.res.web.hosting-environment.yml

```diff
@@ -0,0 +1,83 @@
+name: "avm.res.web.hosting-environment"
+
+on:
+  schedule:
+    - cron: "0 12 1/15 * *" # Bi-Weekly Test (on 1st & 15th of month)
+  workflow_dispatch:
+    inputs:
+      staticValidation:
+        type: boolean
+        description: "Execute static validation"
+        required: false
+        default: true
+      deploymentValidation:
+        type: boolean
+        description: "Execute deployment validation"
+        required: false
+        default: true
+      removeDeployment:
+        type: boolean
+        description: "Remove deployed module"
+        required: false
+        default: true
+
+  push:
+    branches:
+      - main
+    paths:
+      - ".github/actions/templates/avm-**"
+      - ".github/workflows/avm.template.module.yml"
+      - ".github/workflows/avm.res.web.hosting-environment.yml"
+      - "avm/res/web/hosting-environment/**"
+      - "avm/utilities/pipelines/**"
```

Comment:
please take a look in the main branch, we have updated the triggers as per #775

Reply:
Added the following in: - "!avm/utilities/pipelines/platform/**"

Thanks @ChrisSidebotham